srcalc/src/main/webapp/WEB-INF/views/fragments/referenceItem.jsp

```diff
@@ -3,7 +3,14 @@
 <%-- This file cannot be a jspf extension because it is not included statically. --%>
 <%-- This jsp expects for displayItemParam to be defined. --%>
 <ul class="patientNote">
-<c:forEach var="item" items="${displayItemParam.referenceInfo}">
-    <li>${item}</li>
-</c:forEach>
+    <c:choose>
```

Comment:
I'm not sure what's going on here, I just checked my code and everything is indented with spaces. It definitely doesn't look the same.